src/_pytest/terminal.py

```diff
@@ -879,16 +883,13 @@ def mkrel(nodeid):
             if self.verbosity >= 2 and nodeid.split("::")[0] != fspath.replace(
                 "\\", nodes.SEP
             ):
-                res += " <- " + bestrelpath(self.startpath, fspath)
+                res += " <- " + bestrelpath(self.startpath, Path(fspath))
```

Comment:
Note: this is actually a bug fix, we are working on it in #9238

Reply:
interesting, thanks

Reply:
#9238 was merged; can you rebase?